libs/ui/src/components/NeoTabs/NeoTabs.scss

```diff
@@ -0,0 +1,111 @@
+@import '../../scss/_theme.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_expressions.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_variables.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_animations.scss';
+@import '@oruga-ui/oruga/src/scss/utilities/_helpers.scss';
+
+$tabs-font-size: 1rem;
+
+@import '@oruga-ui/oruga/src/scss/components/_tabs.scss';
+
+.neo-tabs--toggle {
+  @include ktheme() {
+    background: theme('background-color');
+    box-shadow: theme('primary-shadow');
+  }
+
+  .o-tabs__content {
+    padding: 0;
```

Comment:
What happen if you remove this line? I don't remember why we remove padding here

Reply:
it will be like this on activity contents:

![Screenshot 2023-07-04 194940](https://github.com/kodadot/nft-gallery/assets/734428/e166a722-b626-45ee-a781-57051244140f)

I forgot to re-add the padding in the content, since the DisabledTab component already uses padding (py-5) https://github.com/kodadot/nft-gallery/blob/0f7cd0a8fb0cf84850fcf74479ffdc7e97839fe8/libs/ui/src/components/DisablableTab/DisablableTab.vue#L2